source/blog/2016-07-29-last-week-in-manageiq-saying-goodbye-to-rubyrep.html.md

```diff
@@ -0,0 +1,122 @@
+---
+title: 'Last Week in ManageIQ: Saying Goodbye to rubyrep'
+author: syncrou
+tags: 
+date: '2016-07-29 11:34:55 EDT'
+comments: true
+published: true
+---
+
+Hello! [Drew](http://gitub.com/syncrou) here, 
+
+I'm taking a breather from [provisioning](https://gitter.im/ManageIQ#provisioning) land to chat briefly with everyone about the
+
+[community](https://github.com/ManageIQ/manageiq/graphs/contributors) accomplishments during an extremely active [*Last Week in ManageIQ*][PRs merged last week].
+
+
+## Featured
+
+On July 27 2016 we bid adieu to the final vestiges of [rubyrep](https://rubygems.org/gems/rubyrep) 
+and now rely soley on [pglogical](https://github.com/2ndQuadrant/pglogical) to handle all of our replication needs.
+
+[#9696](https://rubygems.org/gems/rubyrep) closed out a chapter and started a new book.
+
+In this story all the people are singing! [Happiness and smiles abound](http://giphy.com/gifs/silicon-valley-3o6EhWeei7wsp2jc1G/tile), balloons and confetti are ...
+
+Well, at least a lot less people are losing sleep at night!
+
+Making the move from rubyrep to pglogical was a giant undertaking. A very huge thank-you to everyone involved!
+
+## Improved
+
+### [Allow for service templates to have multiple VM's](https://github.com/ManageIQ/manageiq/pull/8515)
+
+When setting `option_1_number_of_vms` in a service dialog we would only provision 1 virtual machine
+regardless of the value passed in.
+
+This improvement reads the `number_of_vms` and exposes that change in the following 4 areas:
+
+*Provision Request*
+
+*Single Service Item Dialog*
+
+*Service Bundle Dialog*
+
+*Default: Allowing a single vm under a service*
+
+### [Applying RBAC search to Automate Simulation](https://github.com/ManageIQ/manageiq/pull/10060)
+
+The Automate Simulation tool includes two dropdowns `Type` and `Selection` in the Object Attribute panel.
+
+The populated data used to allow a full object selection regardless of who was logged in.
+
+This change locks the list down via RBAC and only shows the user what they are allowed to see.
+
+## Fixed
+
+### [Moved trees on group summary and editor out of session hash](https://github.com/ManageIQ/manageiq/pull/10099)
+
+The existence of trees in session was causing errors when a user had a large number of tags in their environment.
+This fix pulls three trees from the Group Summary screen out of session and stores the data in instance variables.
+
+
+### [Only migrate rows in the current region](https://github.com/ManageIQ/manageiq/pull/10124)
+
+We found an issue with replication when rows which belonged to a region were migrated.
+The join table received an id in the global region when it should have had a remote region
+id assigned.
+
+`cloud_subnets_network_ports` prior to the change:
+
+`vmdb_production=# select * from cloud_subnets_network_ports;
```

Comment:
It looks like this isn't rendering very well - is there a way it could be formatted in markdown?